tests/utils/shippable/shippable.sh

```diff
@@ -148,6 +148,8 @@ ansible-galaxy collection install google.cloud -p "${COLLECTION_DIR}"
 ansible-galaxy collection install community.kubernetes -p "${COLLECTION_DIR}"
 ansible-galaxy collection install f5networks.f5_modules -p "${COLLECTION_DIR}"
 
+chmod -R a+rX "${COLLECTION_DIR}"
```

Comment:
What is this needed for?
I wonder if we need it in other collections which use Shippable.

Reply:
I noticed that several tests fail because they try to include some Python module from another collection, but that failed because of permission errors. I could reproduce this locally: when I installed something with galaxy, often the files had mode `0600` (directories had mode `0755`), and `ansible-test --docker` also produced the same import failures. When I did a `chmod -R a+rX` first, the files also had mode `0644` in the container and suddenly the corresponding unit tests passed.

I think this is more like a bug in ansible-test, it should make sure that the files copied into the docker container are readable by the user used to run the tests (the files are owned by `root`, and apparently another user is used to execute them). But for now, the quickest solution is to simply make everything world-readable before ansible-test is executed.

Reply:
I wonder if that's the problem with galaxy tarballs 

https://github.com/ansible/ansible/issues/66803

Reply:
I'm not sure. ansible-test should still make files readable to the users it uses in the docker containers. Potentially users of ansible-test make sure all their local files are not world-readable. Then they will suddenly have problems with ansible-test if they use the `--docker` option. That's against the principle of least surprise: if you run ansible-test with a user which can access the files, you would expect that the tests can also access these files.

Reply:
Felix: Thank you for the background

@mattclay @jborean93 Do you have any thoughts on the above discussion regarding file permissions and Docker?

Reply:
That looks like an issue with content installed from collections. I don't know why installed content has 0600 permissions.

Reply:
Could it be that the umask in Shippable is per default `0077` and not `0022`?

Reply:
Ok, just checked, `ansible-galaxy collection install` also does this when the umask is `0022`.

Reply:
This is probably caused by https://github.com/ansible/ansible/blob/devel/lib/ansible/galaxy/collection.py#L1035-L1051 (temp files should better be `0600`, and the result is moved to the destination without its mode being adjusted).